cpu/stm32/include/cpu_conf.h

```diff
@@ -123,6 +123,7 @@ extern "C" {
 #if defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4) || \
     defined(CPU_FAM_STM32F7)
 #define PERIPH_FLASHPAGE_CUSTOM_PAGESIZES
+#define PERIPH_FLASHPAGE_NEEDS_FLASHPAGE_ADDR
```

Comment:
Mind adding some doxygen for this to `drivers/include/periph/flashpage.h`? Similar to how it is done for `PERIPH_FLASHPAGE_CUSTOM_PAGESIZES`

Reply:
added